### ISSUE TITLE
Armor (drones) not higher than 8

### ISSUE THREAD

**Issue by wondalfo** [chummer5a/chummer5a#4644]
**Describe the bug**
Whenever chosing a drones armor modification higher than 8, anly an armor modification of 8 is applied. (R5 drones modifications enabled)
 
**To Reproduce**
1. Add a Nissan Doberman
2. Add a modification to this drone
3. choose armor (drones)
4. Choose upgrade level 9 or higher
5. Apply
6. Only an armor update of 8 is in the list and also only the Nuyens for this are listed

**Expected behavior**
The choosen update level should be applied

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 5.216.0.0]

**Comment by chummer5a** [comment]
Working as intended. 
>R5 123: The one thing to remember is that a drone’s attributes can never by higher than twice their starting value (so if you had a Handling of 3, it can never be modifed higher than 6).

**Comment by wondalfo** [comment]
I don't think that you are right here. 
In the german book (the only one I have) the rule you cited is on page 129. On the same page is the explanation for amor modiufication. Their is explicitly written, that a drone can carry amor of three times of its hull (Rumpf) valure, without any reductions in its function. For exery three points over that limit (3x hull) handling and speed is reduced by one. Additionally for every six points accelaration is reduced by 1.

As it is their so explicitly written, I don't think that amor is included in the rule you cited.

Additionally there is a GM-Nissan Dobermann with armor 9 in the german R5 on page 148. I will write an feature request to include this.

I would be happy, if you could comment on this.

Best regards Wolfgang

**Comment by chummer5a** [comment]
Actually disregard that previous, you're right. I can't replicate the issue though. Are you trying to add Armor (Standard) or Armor (Drone)?

**Comment by wondalfo** [comment]
In character settings: R5: yes; drone modification rules: yes; custom date: none selected

1.     Add a Nissan Doberman --> armor 4
2.     Add a modification to this drone
3.     choose armor (drones)
4.     Choose upgrade level 9 or higher
5.     Apply
6.     Only an armor update of 8 is in the list and also only the Nuyens for this are listed

I added Armor (Drone). I could reproduce it, right now. 

I found the german Doberman with armor 9 for 8000N by checking "german data changes" in Custom Data. So my issue is solved in a different way.
